lua/curl/buffers.lua

```diff
@@ -40,6 +40,8 @@ local function open_or_goto_curl_tab()
 
 	if open_with == "split" then
 		vim.cmd("botright split | wincmd j")
+	elseif open_with == "vsplit" then
+		vim.cmd("vsplit | wincmd l")
```

Comment:
Maybe `vim.cmd("botright vsplit | wincmd l")`? This would make behaviour consistent with horizontal split, where it will split the entire screen, and not just the current pane.